src/sdk/pynni/nni/smartparam.py

```diff
@@ -47,39 +47,39 @@
 
 if trial_env_vars.NNI_PLATFORM is None:
     def choice(*options, name=None):
-        return random.choice(options)
+        return param_exp.choice(options, np.random)
```

Comment:
maybe better to use 'np.random.RandomState()' to create an object other than 'np.random'